custom/templates/DefaultRevamp/index.tpl

```diff
@@ -65,9 +65,11 @@
                 </div>
             </div>
             {foreachelse}
-            <div class="ui segment red">
-                {$NO_NEWS}
-            </div>
+                {if $FORUM_ENABLED}
```

Comment:
This is pretty cursed - ideally the index file should have no idea about what modules are enabled. This can be fixed later:tm: with the module home page type revamp

Reply:
Why don’t you just check if the variable is set?